### DISCUSSION TITLE
Pressure Sensitivity Curve Setting for Apple Pencil

### DISCUSSION THREAD

**Discussion by BoroseMedia** [keiwando/toonsquid#167] [Ideas]
Will future updates of the Apple Pencil include pressure sensitivity settings? This feature allows users to adjust the pressure curve of the Apple Pencil. Currently, I must apply a significant amount of pressure to achieve my desired brush strokes. Excessive pressure can cause premature wear on the Apple Pencil tip and screen protector, as well as strain on the hands and wrists. Without this feature, I feel the need to turn to third party apps for drawing everything. I’d love to see this implemented in the future.

![67C545B2-B209-4EC3-B6DB-B29FA1B8E88C](https://user-images.githubusercontent.com/126034792/220501046-22bb8b42-bfc7-4a9c-99b4-8b90b13835e2.jpeg)
![1C6D35AC-B946-4EDE-92A1-F19299C87E8F](https://user-images.githubusercontent.com/126034792/220501062-3af80de8-de8d-453c-8ebf-62836035bfaf.jpeg)

**Comment by dreeze12** [comment]
up

**Comment by 6amane6** [comment]
Apologies for the necro bump but this is also a feature I'd like to see. I've been trying to replicate a favorite brush from Clip Studio and I can't reach my desired pressure. For reference, I've attached a photo. The first line is done with my finger. The second is the pressure I normally draw at, the third is the maximum possible pressure I would feel comfortable applying, and the fourth is grinding down on my screen in a way that felt really unnatural. The bisecting line shows that I couldn't reach 100% opacity on the second line nor could I  quite reach it on the third either, and I'd really like to be able to do that without pressing down so hard on the pen.

![IMG_6597](https://github.com/keiwando/toonsquid/assets/1109363/1c1a9f47-eec8-4401-aede-5677d90d704c)

This issue aside, I want to send my support and compliments for the app, as I'm enjoying it a lot!

**Comment by DLudo** [comment]
I was thinking exactly the same. Just bought the app last week and while i find it an amazing opportunity to drop Procreate or Adobe fresco because of it's advanced animation features i wish it could have the same feeling of drawing as these two apps. Escpecially for the basic pencil

**Comment by ota-works** [comment]
Dude, Is that your art? Could I get a link to your instagram or youtube channel?
